### PR TITLE
Added user choice heartbeat bar

### PR DESCRIPTION
**Pull request by Ponkhy**
As mentioned in #206 was it a little hard to read the Monitor Name on smaller screens.
So i added the possibility that the user can choose how the heartbeat bar should look like.

**Settings Page:** 
Settings

**Example Normal:**
Normal

**Example Bottom:**
Bottom

**Example None:**
Node

### CODE REVIEW

**Comment by Ponkhy** [comment]
I also added a bit more space over the badge on mobile screens to it would look a more centered.

File: src/mixins/theme.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default {
         return {
             system: (window.matchMedia("(prefers-color-scheme: dark)").matches) ? "dark" : "light",
             userTheme: localStorage.theme,
+            userHeartbeatBar: localStorage.heartbeatBarTheme
```
Comment (Saibamen) [inline src/mixins/theme.js:7]:
```suggestion
            userHeartbeatBar: localStorage.heartbeatBarTheme,
```

File: src/mixins/theme.js [hunk 2]
```diff
@@ -13,6 +14,11 @@ export default {
             this.userTheme = "light";
         }
 
+        //Default Heartbeat Bar
```
Comment (Saibamen) [inline src/mixins/theme.js:17]:
```suggestion
        // Default Heartbeat Bar
```

File: src/mixins/theme.js [hunk 3]
```diff
@@ -13,6 +14,11 @@ export default {
...
+        if (! this.userHeartbeatBar) {
```
Comment (Saibamen) [inline src/mixins/theme.js:18]:
```suggestion
        if (!this.userHeartbeatBar) {
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Comment by Ponkhy** [comment]
@Saibamen It should be done

**Review by Saibamen — APPROVED** [review]

**Comment by louislam** [comment]
Found a strange bug, it only appears if your switch to another page and back to "Settings" page.

https://user-images.githubusercontent.com/1336778/129719974-76debf13-ba8a-42dc-84c1-09459319cab8.mp4

**Comment by Ponkhy** [comment]
> Found a strange bug, it only appears if your switch to another page and back to "Settings" page.
> 
>  a.mp4

That's really strange, in the last seconds (0:11-1:12) you can even see a bit of the heartbeat bar.
I'll try to find some time later to figure it out.

**Comment by Ponkhy** [comment]
@louislam Okay it's fixed with the latest commit `77369bd`. I was only able to get it work by adding a `:key`.
I've also used `v-show` instead of `v-bind:class` now and removed the `.hideHeartbeatBar` class.

**Pull request merged**